### PR TITLE
feed.rss  feed.json

### PR DESCRIPTION
**Pull request by tyrelsouza**
`/feed.rss`

```xml
<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
  <channel>
    <title>Conda recent packages</title>
    <description>Recent Conda packages</description>
    <link>http://conda.libraries.io</link>
    <item>
      <title>conda-standalone</title>
      <link>http://conda.libraries.io/package?name=conda-standalone</link>
      <description>conda-standalone updated on </description>
      <channel>pkgs/main</channel>
      <pubDate>1568729357</pubDate>
    </item>
    <item>
      <title>plotly</title>
      <link>http://conda.libraries.io/package?name=plotly</link>
      <description>plotly updated on </description>
      <channel>pkgs/main</channel>
      <pubDate>1568404033</pubDate>
    </item>
  </channel>
</rss>
```

`/feed.json`
```json
["conda-standalone", "plotly"]
```

### CODE REVIEW

File: conda.rb [hunk 1]
```diff
@@ -66,10 +66,30 @@ def download_and_parse_packages
     channels_with_packages
   end
 
+  def latest(count)
+    packages = @redis.scan_each(match: "packages:*").to_a.map do |key|
```
Comment (katzj) [inline conda.rb]: How many packages are there in total for conda? It may make sense to add a sorted set from the job also to avoid having to do the scan.

Comment (tyrelsouza) [reply conda.rb]: 1783

File: app.rb [hunk 2]
```diff
@@ -23,6 +23,11 @@ class CondaAPI < Sinatra::Base
     end
   end
 
+  get '/feed.rss' do
```
Comment (mikeyoung85) [inline app.rb:34]: Could we also add a JSON endpoint for these?

File: conda.rb [hunk 3]
```diff
@@ -72,4 +84,13 @@ def download_channeldata(channel, domain)
     url = "https://#{domain}/#{channel}/channeldata.json"
     HTTParty.get(url).parsed_response
   end
+
+  def channel_and_name_from_key(key)
+    # Split apart the channel and name from the redis key `packages:channel/name`
+    channel_name = key.rpartition(":").last.rpartition("/")
```
Comment (mikeyoung85) [inline conda.rb]: You can condense this down to one line. Something like `channel, _separator, name = key.rpartition(":").last.rpartition("/")`

Comment (tyrelsouza) [reply conda.rb]: ah right because rpartition only does two splits,  derp

**Review by mikeyoung85 — APPROVED** [review]
Looks good to me

**Pull request merged**